acvm_backend_barretenberg/src/composer.rs

```diff
@@ -74,24 +85,32 @@ impl StandardComposer {
         result
     }
 
-    pub fn compute_verification_key(&self, proving_key: &[u8]) -> Vec<u8> {
+    fn compute_verification_key(
+        &self,
+        constraint_system: &ConstraintSystem,
+        proving_key: &[u8],
+    ) -> Vec<u8> {
+        let circuit_size = self.get_circuit_size(constraint_system);
+        let CRS {
+            g1_data, g2_data, ..
+        } = CRS::new(circuit_size as usize);
```

Comment:
Does this actually use g1_data? Because it might be more efficient to use a `G2::new` instead